### DISCUSSION TITLE
Can I add the ability to delete files in the github mobile？

### DISCUSSION THREAD

**Discussion by 873578156** [community/community#109754] [Mobile]
Can I add the ability to delete files. think s!

**Comment by cy-Yin** [comment]
You can get the answers [here](https://docs.github.com/en/repositories/working-with-files/managing-files/deleting-files-in-a-repository) in GitHub Docs.

If you want to use **git** to remove files, these two from [StackOverflow](https://stackoverflow.com/questions/2047465/how-do-i-delete-a-file-from-a-git-repository) and [GitHub Docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository) may be helpful for this problem.

Firstly make sure that you have the write permissions in the repo.

**Reply by cy-Yin** [reply]
If you want others to have the write permission to a repo, make sure they are your organization members according to this [article](https://docs.github.com/en/organizations/managing-organization-settings/setting-permissions-for-deleting-or-transferring-repositories).

**Comment by Shiny0805** [comment]
In generally, github doesn't provide a built-in feature to delete files directly through the user interface of the website.
Anyhow, You can delete from a Github repository using Git commnads on your local machine and then pushing those changes to the remote repository.

`git rm filename.txt`
`git commit -m "Delete filename.txt"`
`git push origin main`

**Reply by nullmedia-social** [reply]
How do I delete files on the mobile app? This is what poster was asking. We can't use commands like that on mobile (iOS in my case, and that applies to iOS especially).
